include/SH3/system/glprogram.hpp

```diff
@@ -26,6 +27,7 @@ namespace sh3_graphics
     public:
 
         sh3_glprogram(const std::string& name) : programName(name){Load(name);};
+        sh3_glprogram(const std::string& name, std::vector<std::string> const &attribs) : programName(name){Load(name, attribs);};
```

Comment:
So should I default them to empty using `std::vector<std::string> const &attribs = {}` in the constructor and method?